entrypoint.sh

```diff
@@ -36,6 +38,14 @@ if [ -z "${SUBSPACE_LISTENPORT-}" ]; then
   export SUBSPACE_LISTENPORT="51820"
 fi
 
+if [ -z "${SUBSPACE_ENDPOINT_HOST-}" ]; then
+  export SUBSPACE_ENDPOINT_HOST="" # Use default values defined in subspace command
+fi
+
+if [ -z "${SUBSPACE_ALLOWED_IPS-}" ]; then
+  export SUBSPACE_ALLOWED_IPS="" # Use default values defined in subspace command
```

Comment:
This needs to be set to `0.0.0.0/0, ::/0` if its not specified as the flag being passed in overrides the `go` default which means it comes out as blank

Reply:
That's right! Thanks.